src/components/Table/Table.tsx

```diff
@@ -252,6 +252,9 @@ interface IThProps
 	/** Sets the width of the cell. */
 	width?: number | string;
 
+	/** Sets the min width of the cell. */
+	minWidth?: number | string;
```

Comment:
Are there any times where we need to convert `minWidth` from a string to number. Similar to what is being done on line 498.